src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.System/Properties/Resources.resx

```diff
@@ -117,6 +117,13 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="confirm_system_commands" xml:space="preserve">
+    <value>Confirm system commands</value>
```

Comment:
Let's rename it to: `Show a dialog to confirm system commands`. This message should be clear to a user.

Reply:
Done